products/monitoring/api.yaml

```diff
@@ -702,10 +702,14 @@ objects:
           entries in this field is
           `projects/[PROJECT_ID]/notificationChannels/[CHANNEL_ID]`
         item_type: Api::Type::String
-      - !ruby/object:Api::Type::Array
-        name: 'labels'
```

Comment:
Do you mind marking this as deprecated using an extra schema entry? It's a breaking change to remove a field, and could give users who specified it a syntax error.

This is a little silly- we know there can't be users because the field doesn't work- but it's little enough effort that I'd rather do it by the book.

https://github.com/GoogleCloudPlatform/magic-modules/blob/master/templates/terraform/extra_schema_entry/backend_service.erb is a good template, replacing `Removed` with `Deprecated`.

Reply:
Done